frontend/src/components/EstimateSection/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Button } from '../shared/button';
+import './style.css';
+
+const EstimateSection = () => {
+  return (
+    <div className="estimate__section_bg py-32">
```

Comment:
please use section instead of div, to apply the global section style. Also for semantics.